content/blog/blog-post-11.md

```diff
@@ -0,0 +1,23 @@
+---
+authors: ["kokkos-team"]
+title: "Field report for the Kokkos User Group Meeting 2025"
+date: 2025-05-09
+tags: ["blog"]
+thumbnail: img/blog/KUG2025-1.jpg
```

Comment:
We don;t have anything better for the front matter image?

Reply:
I am good with it.